templates/Reference.md

```diff
@@ -16,22 +16,29 @@ As of v2.10+ the label argument is optional. When the label is not passed, no en
 set size [uint32]
 ```
 
-| Type | Description |
-| --------| ----------- |
-| uint64  | Reads an unsigned 64-bit integer |
-| int64   | Reads a signed 64-bit integer |
-| uint32  | Reads an unsigned 32-bit integer |
-| int32   | Reads a signed 32-bit integer |
-| uint16  | Reads an unsigned 16-bit integer |
-| int16   | Reads a signed 16-bit integer |
-| uint8   | Reads an unsigned 8-bit integer |
-| int8    | Reads a signed 8-bit integer |
-| float   | Reads a 32-bit floating point |
-| double  | Reads a 64-bit floating point |
-| uuid    | Reads 16-byte UUID |
-| macdate | Reads classic Mac OS 4-byte date (seconds since January 1, 1904) |
-| fatdate | Reads FAT, or DOS, 2-byte date (v2.13+) |
-| fattime | Reads FAT, or DOS, 2-byte time (v2.13+) |
+| Type        | Description |
+| ------------| ----------- |
+| uint64      | Reads an unsigned 64-bit integer |
+| int64       | Reads a signed 64-bit integer |
+| uint32      | Reads an unsigned 32-bit integer |
+| int32       | Reads a signed 32-bit integer |
+| uint24      | Reads an unsigned 24-bit integer |
+| uint16      | Reads an unsigned 16-bit integer |
+| int16       | Reads a signed 16-bit integer |
+| uint8       | Reads an unsigned 8-bit integer |
+| int8        | Reads a signed 8-bit integer |
+| uint8_bits  | Read some number of `bits` out of an unsigned 8-bit integer |
```

Comment:
I think we should have the uintx_bits functions as their own section since this doesn't document how to use them. you can leave them off of this PR if you don't want to expand on that for now.

Reply:
The goal was to get them in since some documentation seemed better than none, but I'm ok with a separate PR for the `_bits` variants.